packages/cpp-debug/package.json

```diff
@@ -25,6 +25,10 @@
     "@theia/debug": "next"
   },
   "devDependencies": {
+    "@types/chai": "^4.1.7",
+    "@types/mocha": "^5.2.0",
+    "chai": "^4.2.0",
+    "mocha": "^5.2.0",
```

Comment:
Might be silly, but these are new dependencies, and I wonder if they too should require a CQ :/